tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -195,6 +265,7 @@ class ImagePreview(Base):
     _gallery_button_locator = (By.CSS_SELECTOR, 'button[data-l10n-id="open-gallery"]')
     _confirm_delete_button = (By.ID, 'dialog-yes')
     _option_menu_locator = (By.CLASS_NAME, 'js-menu')
+    _not_video_preview_locator = (By.CSS_SELECTOR, '[style="display: none;"]')
```

Comment:
This selector looks far too generic to me. It will select anything with this style.